pkg/client/secrets.go

```diff
@@ -53,6 +54,7 @@ func (c *DefaultSecretsClient) CreateSecret(ctx context.Context, organizationID
 		XDispatchOrg: c.getOrgID(organizationID),
 		Secret:       secret,
 	}
+	log.Infof("creating secret %s/%s", organizationID, *secret.Name)
```

Comment:
Some debug leftovers?

Reply:
I decided to leave it in... I can remove

Reply:
done